backend/site-api/src/app/admin/reset_all.rs

```diff
@@ -83,6 +89,13 @@ pub async fn reset_all(
                     },
                 )
                 .await?;
+
+                create_custom_table_helper(
+                    &context,
+                    &site_id,
+                    serde_json::from_str(SITE_SEED_CONTACT_TABLE).unwrap(),
```

Comment:
Currently, all three sites are seeded with the same table. If necessary, we can create different data for testing.

Reply:
I think it's fine for now 👍